cli/src/main/resources/hudson/cli/client/Messages.properties

```diff
@@ -14,7 +14,8 @@ CLI.Usage=Jenkins CLI\n\
    -logger FINE : enable detailed logging from the client\n\
    -auth [ USER:SECRET | @FILE ] : specify username and either password or API token (or load from them both from a file);\n\
                                    for use with -http, or -remoting but only when the JNLP agent port is disabled.\n\
-                                   Not necessary if JENKINS_USER_ID and JENKINS_API_TOKEN env vars are set\n\
+                                   Passing crendentials by a file is recommended.\n\
+                                   See https://jenkins.io/doc/book/managing/cli/#http-connection-mode for more info and options.\n\
```

Comment:
Should reference a redirect URL  to be future-proof.

Reply:
Created https://github.com/jenkins-infra/jenkins.io/pull/1816/files